### DISCUSSION TITLE
Fees

### DISCUSSION THREAD

**Discussion by Black1Matter** [cosmos/community#57] [Chain Builders (Cosmos SDK)] [answered]
can you still tell me where the fees we spend on creating transactions go, they don't just disappear?
Maybe you know?
is it shared between validators only?

**Comment by julienrbrt** [comment, marked as answer]
Hi, all the fees are collected and redistributed to validators and stakers.
They are sent to the fee_collector module account (https://github.com/cosmos/cosmos-sdk/blob/f90d7e145594fef96e8092d2dcf4b625b12ebf8b/x/auth/types/keys.go#L15) and then redistributed. On the Cosmos Hub, it translates to this address: https://www.mintscan.io/cosmos/address/cosmos17xpfvakm2amg962yls6f84z3kell8c5lserqta
